### DISCUSSION TITLE
The podcast namespace proposal -> adoption process

### DISCUSSION THREAD

**Discussion by Dwev** [Podcastindex-org/podcast-namespace#615] [General]
Sorry if this is covered somewhere, but I was just wanting some more information on the process of a proposal going into a phase, and then getting adopted (i.e. governance.) I'm new to this kind of discussion and want to make sure I contribute in the best way.

- If there are 50 new tags in the proposal "backlog", how is it decided which ones to include in which phase?
- If there are tag enhancements, how is it decided to include an enhancement in a phase discussion?
- How is it decided which tags will actually be adopted for release into the spec for a phase?  Is there voting? How does that work?

**Comment by picard102** [comment]
Agreed, there needs to be more clarity on how this project is governed and what the process for proposal adoption is.

This has some answers you're maybe looking for though. https://github.com/Podcastindex-org/podcast-namespace#current-roadmap

**Comment by samsethi** [comment]
From my point of view as the app with the most tag support (28 tags) I look at all the proposals and I decide if I want to include them in TrueFans or not.  I am a firm believer in 'build fast and fix things'.  With many P2 tags, I have been the first to implement them and then wait for feedback from users and other community developers before adapting them.  I find this works.  

The process of formal adoption is then down to one host and two apps adopting a new tag before it is formally added to a new phase.

**Reply by picard102** [reply]
With my one client we only support 11 tags in their feeds.
